components/clientComponents/forms/Form/Form.tsx

```diff
@@ -216,20 +227,30 @@ const InnerForm: React.FC<InnerFormProps> = (props) => {
           >
             {children}
 
-            <RichText>
-              {form.privacyPolicy &&
-                form.privacyPolicy[props.language == "en" ? "descriptionEn" : "descriptionFr"]}
-            </RichText>
+            {showIntro && (
+              <RichText>
+                {form.privacyPolicy &&
+                  form.privacyPolicy[props.language == "en" ? "descriptionEn" : "descriptionFr"]}
+              </RichText>
+            )}
+
+            {allowGrouping && currentGroup === LockedSections.REVIEW && <Review />}
+
             {props.renderSubmit ? (
               props.renderSubmit({
                 validateForm: props.validateForm,
                 fallBack: () => {
                   return (
-                    <SubmitButton
-                      numberOfRequiredQuestions={numberOfRequiredQuestions}
-                      formID={formID}
-                      formTitle={form.titleEn}
-                    />
+                    <div>
+                      {allowGrouping && currentGroup === LockedSections.REVIEW && <BackButton />}
```

Comment:
@timarney I think we only want the back button in the fallback and NOT in the other case?

Reply:
yes as discussed let's a the url to check if this is a "group" form